src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -121,14 +121,22 @@ public abstract class XmlSerializerImplementation
     /// </devdoc>
     public class XmlSerializer
     {
+#if uapaot
```

Comment:
Any good reason to ifdef this as opposed to just making it public in all builds?